src/amo/sagas/collections.js

```diff
@@ -166,43 +169,56 @@ export function* addAddonToCollection({
   }
 }
 
-export function* updateCollection({
-  payload: {
-    errorHandlerId,
+export function* modifyCollection(
+  action: 'create' | 'update',
+  params: Object,
```

Comment:
I know this should be something more specific than `Object`, but I couldn't figure out what to make it so that Flow was happy, so some advice would be welcome.